client/me/purchases/manage-purchase/purchase-meta-expiration.tsx

```diff
@@ -18,9 +18,9 @@ import {
 } from 'calypso/lib/purchases';
 import { getCurrentUserId } from 'calypso/state/current-user/selectors';
 import AutoRenewToggle from './auto-renew-toggle';
+import type { SiteDetails } from '@automattic/data-stores/src/site/types';
```

Comment:
You should be able (I hope?) to import this directly from the package's main export. Reaching inside of a package by specifying a file path is risky because that's considered part of the package's internals so it could change at any point.

Try this:

```suggestion
import type { SiteDetails } from '@automattic/data-stores';
```